baus.py

```diff
@@ -162,311 +170,281 @@ def run_models(mode, run_setup, years_to_run):
 
     elif mode == "simulation":
```

Comment:
We should really move all the run lists to a separate module, or config file, out of baus.py